GCMessengerSDKSample/app/build.gradle

```diff
@@ -20,8 +20,8 @@ android {
         applicationId = "com.genesys.cloud.messenger.sample"
         minSdk = 21
         targetSdk = 34
-        versionCode = 57
-        versionName = "1.10.2"
+        versionCode = 58
+        versionName = "1.10.3"
```

Comment:
As discussed we it needs to be reverted back to "1.10.2".
Please build Sample App as "1.10.2.rc4".

Reply:
@mobichel I changed the versions please check